Utils/merge_content_new_zip.py

```diff
@@ -0,0 +1,170 @@
+import zipfile as z
+import sys
+import requests
+import os
+from google.cloud import storage
+import google.auth
+import warnings
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+ACCEPT_TYPE = "Accept: application/json"
+CONTENT_API_URI = "https://circleci.com/api/v1/project/demisto/content"
+FILES_TO_REMOVE_LIST = ['content-descriptor.json', 'doc-CommonServer.json', 'doc-howto.json']
```

Comment:
@yaakovi you are better familiar with this then i am, anything else we should add here?

Reply:
@barchen1 i think that maybe we should include the old `'doc-CommonServer.json', 'doc-howto.json'` too, as they will be needed for old versions

Reply:
special files:
* `content-descriptor.json` take only the new one (need to think of a way to include the asset ID (build number) of the secondary zips
* `doc-CommonServer.json` - take only the new (no reason to support the old)
* `doc-howto.json` - take only the new one (no reason it will be changed in older versions)
* `reputations.json` - there should only be one, need to merge both of them (in case there is an update)